locales-lt-LT.xml

```diff
@@ -77,6 +75,7 @@
 
     <!-- ORDINALS -->
     <term name="ordinal">-asis</term>
+    <term name="ordinal-03">-iasis</term>
```

Comment:
You can specify "ordinal-13" to override this further for the 13 case. I think it was worthwhile to keep the feminine ordinals. It may not be used currently, but it doesn't hurt to have it in here.